### ISSUE TITLE
Price and Status filters don't get reapplied

### ISSUE THREAD

**Issue by StilyanMitrev** [EnterDAO/LandWorks-FE#812]
At the moment when I apply a price or a status filter, the cards are getting filtered, however if I click in a card, or go to my properties after that and come back to the Explore page, the filter buttons indicate that the filter is on but the filter is not applied. 

Example: I go in the Explore page -> I change the Price filter to only ETH with Max of 0.01 -> I apply the filter. -> I click on a property -> go back to the Explore page -> the filter is not applied but the price filter button shows that the filter is active. 

Basically the Status and Price filters are not getting applied when you click out of the Explore page and come back.

As per @n0vax's idea - include the filter setting in the URL

AC (Acceptance Criteria): Given I apply a price or a status filter in the Explore page and then I click away from the Explore page going in My Properties or in Property Details and then I come back to the Explore page then the applied price or status filter should be active.

**Comment by StilyanMitrev** [comment]
Neither price, nor status filters is active once i move away from explore
